src/test/kotlin/fi/metatavu/timebank/api/test/functional/data/TestData.kt

```diff
@@ -52,6 +52,21 @@ class TestData {
             }
         }
 
+        /**
+         * Gets mock Forecast API response to /v4/time_registrations
+         *
+         * @return ForecastTimeEntryResponse - 1 entry
+         */
```

Comment:
Add something to the documentation about how this is used for testing the deletion synchronization.